ui5-task-zipper error when zipping with ushell library added

## Expected Behavior
Can run the ui5-task-zipper as part of the build process without it failing, when the ushell library is added

## Current Behavior
when running the ui5-task-zipper if the library ushell has been added the following error occurs, preventing the creation of the zip until that library is removed. This was previously working however I updated my UI5 version and UI5 CLI versions today, so I assume it is one of those which has caused this.

"Couldn't add all resources to the archive: Error: absolute path: /test-resources/sap/ushell/bootstrap/sandbox.js"

## Steps to Reproduce the Issue
1. In your ui5.yaml add the library ushell (it does not matter if it is flagged as a development only library, the behaviour is the same)
2. Also add the ui5-task-zipper 
3. Attempt to build the UI5 app, and receive the error as described
4. Remove the ushell module and build again and it builds successfully

## Context
 * SAPUI5 Version: **`1.84.3`**
 * UI5 Module Version (output of `ui5 --version` when using the CLI): **`2.7.0`**
 * Node.js Version: **`12.16.1`**
 * npm Version: `6.14.8`
 * OS/Platform: `MacOS Catalina 10.15.7`
 * Browser *(if relevant)*: `NA`
 * Other information regarding your environment *(optional)*: `I am using nvm, although this has never mattered in the past`

## Workaround?
Yes, the workaround is to either remove the ushell library when building for deployment, or use a separate config file for developing and building

## ui5.yaml file
```
specVersion: "2.2"
metadata:
  name: capture
type: application
resources:
  configuration:
    paths:
      webapp: webapp
framework:
  name: SAPUI5
  version: 1.84.3
  libraries:
    - name: sap.ui.core
    - name: sap.m
    - name: sap.ui.layout
    - name: sap.ushell
      development: true
    - name: themelib_sap_fiori_3
server:
  customMiddleware:
    - name: ui5-middleware-livereload
      afterMiddleware: compression
      configuration:
        port: 35729
        path: webapp
    - name: ui5-middleware-simpleproxy
      mountPath: /v2/example
      afterMiddleware: compression
      configuration:
        baseUri: http://localhost:4004/v2/example
builder:
  resources:
    excludes:
      - "/test/**"
      - "/test-resources/**"
  customTasks:
    - name: "ui5-task-zipper"
      afterTask: "uglify"
      configuration:
        archiveName: "captureui"
        keepResources: true
```

## Log Output / Stack Trace
```
npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/Users/<removed>/.nvm/versions/node/v12.16.1/bin/node',
npm verb cli   '/Users/<removed>/.nvm/versions/node/v12.16.1/bin/npm',
npm verb cli   'run',
npm verb cli   'build:ui',
npm verb cli   '--verbose'
npm verb cli ]
npm info using npm@6.14.8
npm info using node@v12.16.1
npm verb run-script [ 'prebuild:ui', 'build:ui', 'postbuild:ui' ]
npm info lifecycle capture@0.0.1~prebuild:ui: capture@0.0.1
npm info lifecycle capture@0.0.1~build:ui: capture@0.0.1

> capture@0.0.1 build:ui /Users/<removed>/dev/NGN/audit-apps/capture
> run-s  build:uimodule

npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/Users/<removed>/.nvm/versions/node/v12.16.1/bin/node',
npm verb cli   '/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/bin/npm-cli.js',
npm verb cli   'run',
npm verb cli   'build:uimodule'
npm verb cli ]
npm info using npm@6.14.8
npm info using node@v12.16.1
npm verb run-script [ 'prebuild:uimodule', 'build:uimodule', 'postbuild:uimodule' ]
npm info lifecycle capture@0.0.1~prebuild:uimodule: capture@0.0.1
npm info lifecycle capture@0.0.1~build:uimodule: capture@0.0.1

> capture@0.0.1 build:uimodule /Users/<removed>/capture
> ui5 build --config=ui5.yaml --clean-dest --dest dist --include-task=generateManifestBundle generateAppPreload

info normalizer:translators:ui5Framework Using SAPUI5 version: 1.84.3
info builder:builder Building project capture not including dependencies...
info builder:builder 🛠  (1/1) Building project capture
info builder:builder application capture 🔨 (1/10) Running task escapeNonAsciiCharacters...
info builder:builder application capture 🔨 (2/10) Running task replaceCopyright...
info builder:builder application capture 🔨 (3/10) Running task replaceVersion...
info builder:builder application capture 🔨 (4/10) Running task generateFlexChangesBundle...
info builder:builder application capture 🔨 (5/10) Running task generateManifestBundle...
info builder:builder application capture 🔨 (6/10) Running task generateComponentPreload...
info builder:builder application capture 🔨 (7/10) Running task createDebugFiles...
info builder:builder application capture 🔨 (8/10) Running task uglify...
info builder:builder application capture 🔨 (9/10) Running task ui5-task-zipper...
ERR! builder:customtask:zipper Couldn't add all resources to the archive: Error: absolute path: /test-resources/sap/ushell/bootstrap/sandbox.js
ERR! builder:builder Build failed in 2.57 s
info builder:builder Executing cleanup tasks...

⚠️  Process Failed With Error

Error Message:
absolute path: /test-resources/sap/ushell/bootstrap/sandbox.js

For details, execute the same command again with an additional '--verbose' parameter
npm verb lifecycle capture@0.0.1~build:uimodule: unsafe-perm in lifecycle true
npm verb lifecycle capture@0.0.1~build:uimodule: PATH: /Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/<removed>/capture/node_modules/.bin:/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/<removed>/capture/node_modules/.bin:/Users/<removed>/.nvm/versions/node/v12.16.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/<removed>/Library/Android/sdk/emulator:/Users/<removed>/Library/Android/sdk/tools:/Users/<removed>/Library/Android/sdk/platform-tools:/Users/<removed>/Library/Android/sdk/cmdline-tools/latest/bin
npm verb lifecycle capture@0.0.1~build:uimodule: CWD: /Users/<removed>/capture
npm info lifecycle capture@0.0.1~build:uimodule: Failed to exec build:uimodule script
npm verb stack Error: capture@0.0.1 build:uimodule: `ui5 build --config=ui5.yaml --clean-dest --dest dist --include-task=generateManifestBundle generateAppPreload`
npm verb stack Exit status 1
npm verb stack     at EventEmitter.<anonymous> (/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
npm verb stack     at EventEmitter.emit (events.js:311:20)
npm verb stack     at ChildProcess.<anonymous> (/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
npm verb stack     at ChildProcess.emit (events.js:311:20)
npm verb stack     at maybeClose (internal/child_process.js:1021:16)
npm verb stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
npm verb pkgid capture@0.0.1
npm verb cwd /Users/<removed>/capture
npm verb Darwin 19.6.0
npm verb argv "/Users/<removed>/.nvm/versions/node/v12.16.1/bin/node" "/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/bin/npm-cli.js" "run" "build:uimodule"
npm verb node v12.16.1
npm verb npm  v6.14.8
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! capture@0.0.1 build:uimodule: `ui5 build --config=ui5.yaml --clean-dest --dest dist --include-task=generateManifestBundle generateAppPreload`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the capture@0.0.1 build:uimodule script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm verb exit [ 1, true ]
npm timing npm Completed in 3441ms

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/john.murrayuk.ey.com/.npm/_logs/2020-12-18T08_34_32_273Z-debug.log
ERROR: "build:uimodule" exited with 1.
npm verb lifecycle capture@0.0.1~build:ui: unsafe-perm in lifecycle true
npm verb lifecycle capture@0.0.1~build:ui: PATH: 
/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/<removed>/capture/node_modules/.bin:/Users/<removed>/.nvm/versions/node/v12.16.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/<removed>/Library/Android/sdk/emulator:/Users/<removed>/Library/Android/sdk/tools:/Users/<removed>/Library/Android/sdk/platform-tools:/Users/<removed>/Library/Android/sdk/cmdline-tools/latest/bin
npm verb lifecycle capture@0.0.1~build:ui: CWD: /Users/<removed>/capture
npm info lifecycle capture@0.0.1~build:ui: Failed to exec build:ui script
npm verb stack Error: capture@0.0.1 build:ui: `run-s  build:uimodule`
npm verb stack Exit status 1
npm verb stack     at EventEmitter.<anonymous> (/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
npm verb stack     at EventEmitter.emit (events.js:311:20)
npm verb stack     at ChildProcess.<anonymous> (/Users/<removed>/.nvm/versions/node/v12.16.1/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
npm verb stack     at ChildProcess.emit (events.js:311:20)
npm verb stack     at maybeClose (internal/child_process.js:1021:16)
npm verb stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
npm verb pkgid capture@0.0.1
npm verb cwd /Users/<removed>/capture
npm verb Darwin 19.6.0
npm verb argv "/Users/<removed>/.nvm/versions/node/v12.16.1/bin/node" "/Users/<removed>/.nvm/versions/node/v12.16.1/bin/npm" "run" "build:ui" "--verbose"
npm verb node v12.16.1
npm verb npm  v6.14.8
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! capture@0.0.1 build:ui: `run-s  build:uimodule`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the capture@0.0.1 build:ui script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm verb exit [ 1, true ]
npm timing npm Completed in 3812ms

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/<removed>/.npm/_logs/2020-12-18T08_34_32_303Z-debug.log
```

--- a/packages/ui5-task-zipper/lib/zipper.js
+++ b/packages/ui5-task-zipper/lib/zipper.js
@@ -49,9 +49,8 @@ module.exports = async function ({ workspace, dependencies, options, taskUtil })
         taskUtil.setTag(resource, OmitFromBuildResult, true);
       }
       return resource.getBuffer().then((buffer) => {
-        debugger;
         isDebug && log.info(`Adding ${resource.getPath()} to archive.`);
-        zip.addBuffer(buffer, resource.getPath().replace(prefixPath, '').replace("/resources", 'resources'));
+        zip.addBuffer(buffer, resource.getPath().replace(prefixPath, '').replace("/resources", 'resources').replace("/test-resources", 'test-resources'));
       })
     }));
     // include the additional files from the project

